core/chains/evm/client/chain_id_sub.go

```diff
@@ -77,6 +77,9 @@ func (c *chainIDSubForwarder) forwardLoop() {
 }
 
 func (c *chainIDSubForwarder) Unsubscribe() {
+	// the error channel must be closed by Unsubscribe
+	defer close(c.err)
```

Comment:
Would it be safer to close from the `forwardLoop()`, so that if this is called twice it never tries to close the chan again?

Reply:
Agreed, done.